content/en/product_analytics/journeys/funnel_analysis.md

```diff
@@ -23,32 +23,87 @@ Funnel analysis helps you track conversion rates across key workflows to identif
+When constructing your funnel, you can add [default attributes][2] (core, device, operating system, geo-location, and user) and [session-specific][3] attributes to analyze the data further. Click the three dots button next to an event and click **Add Filters** button to view the full list of available attributes.
 
 {{< img src="real_user_monitoring/funnel_analysis/funnel-analysis-filtering.png" alt="Use attributes to filter information when constructing your funnel" style="width:80%;" >}}
 
+### Combining events
+
+When constructing your funnel, you can combine multiple events within a given step, as end users may have different ways to achieve the same outcome through different flows.
+
+To combine an event, click the three dots next to an event and select **+ Combine Events**.
+
+## Refine conversion
+
+You can further analyze the information on the funnel page to understand the conversion rate. Conversion rate is a crucial metric that measures the effectiveness of your site or application.
+
+You can measure conversion by the following attributes:
+
+- **Session count** or **users** - Understand how many sessions or users ended up completing the funnel.
+- **Conversion count** - A count of users who went through the funnel you've defined.
+- **Conversion rate** - A conversion refers to the moment when a user responds to a call to action. This rate is the percentage of the total number of users that visited your site or application who converted.
+
+  **Note**: Only completed RUM sessions contribute to conversion rates.
+- **Drop-off rate** - A drop-off who left the page or app without finishing a process. This rate is the percentage of the total number of users who visited your site or application who did not complete the last step event.
+- **Time to convert** - The time it took for the user to complete the step events.
+
+You can measure these attributes **across all steps** or between **specific steps**.
+
+{{< img src="product_analytics/journeys/funnel_analysis/funnels-across-steps.png" alt="Measure attributes across all steps or specific steps." style="width:60%;" >}}
+
+Use the **filter** selector to filter by various criteria that you define.
+
+From there, you can click a datapoint to **investigate the specific attributes** that may have affected conversion rates, such as page load speed, ease of navigation, or checkout experience.
+
+## Changing the visualization
+
+{{< img src="product_analytics/journeys/funnel_analysis/funnel-timeseries.mp4" alt="Click the visualization dropdown to select a different view" video=true >}}
+
+After you've defined the step events and conversion measurement, you can switch to see a different visualization and better understand conversion of users for your app.
```

Comment:
```suggestion
After you've defined the step events and conversion measurement, you can switch to a different visualization to better understand user conversions for your app.
```

Reply:
minor suggestion. the "better understand conversion of users for your app" just sounded awkward when I read it out loud.